src/components/attendee/Popover.tsx

```diff
@@ -6,20 +6,33 @@ import { POPOVER_CONTAINER_ID } from "settings";
 
 export interface PopoverProps {
   referenceElement?: Element | null;
+  offset?: number[];
 }
 
 export const Popover: React.FC<PopoverProps> = ({
   children,
   referenceElement,
+  offset,
 }) => {
   const popoverContainerElement = document.querySelector(
     `#${POPOVER_CONTAINER_ID}`
   );
+  const [left, top] = offset ?? [];
 
   const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);
   const { styles: popperStyles, attributes: popperAttributes } = usePopper(
     referenceElement,
-    popperElement
+    popperElement,
+    {
+      modifiers: [
+        {
+          name: "offset",
+          options: {
+            ...(offset && { offset: [left, top] }),
+          },
```

Comment:
or maybe it'd be even better to write something like
```suggestion
          options: { offset },
```

and in the props list:
```ts
offset = ALWAYS_EMPTY_ARRAY,
```

Reply:
Doing it this way results in TS error, that might be solved by importing the `Offset` from `popover.js`, which itself feels incorrect as this lib doesn't provide its own types, and we'd need to create some type guards, maybe use `as` and etc, so I think we'd better off leaving this piece as it is

Reply:
@sunny-viktoryia @alexey-petrov It looks clunky, but this is the way we managed to make it work in the older .js files and is more or less compact even now.

The issue isn't just that TS complains, but that at times missing value i.e. `undefined` shouldn't override an existing one.

I agree there is a nicer way of doing this, but so far we haven't had a time/priority to make some utility functions that may just as clunky as:
```ts
  options: { offset: set( options.offset, offset )  }
```
or
```ts
  setIfProvided( options, 'offset`, offset )
```

Reply:
@goran-peoski-work I've gotten rid of this Popover usage completely
It was at the very least unreliable
 [3f9901e](https://github.com/sparkletown/sparkle/pull/3008/commits/3f9901e443688e07fd70127f60e27974bc7975e4)